### ISSUE TITLE
wing terms requested

### ISSUE THREAD

**Issue by bevmatthews** [FlyBase/drosophila-anatomy-developmental-ontology#385]
For new term requests, please provide the following information:

## Preferred term label

ventral radius
proximal dorsal radius
proximal ventral radius

Comment
We have 

sensillum campaniformium of proximal dorsal radius
sensillum campaniformium of proximal ventral radius
sensillum campaniformium of ventral radius Sc3
etc.
We have multiple terms of type 
sensillum campaniformium of [X] where X is not a term or a synonym in the CV.
I would argue that anything that appears in the position [X] should be a term in the CV at least as a synonym.

**Comment by Clare72** [comment]
Agreed.
Do you have a reference for this? It looks as though these could be synonyms for wing veins - "dorsal radius" is already a synonym for FBbt:00004761 "wing vein L3" (no reference given), but it may be tricky to work out which is which.

**Comment by bevmatthews** [comment]
The reference that led to the question is FBrf0208237. I think you are right that they are synonyms for wing veins. I did a limited search and had a hard time working out what was what. Sorry not to be or more help.

**Comment by Clare72** [comment]
I have added a term for the radius ('radial vein'), as this did not appear to exist in the ontology in the context that it is used in the sensilla papers.
I also made proximal, medial and distal regions of this to help better classify the sensilla.
I didn't make specific terms for the dorsal and ventral parts of each vein (I think this could quickly get out of hand if done for every wing vein), but have tried to clarify where the sensillum is found on the vein in each description.
